extension/public/popup.js

```diff
@@ -101,10 +101,10 @@ const renderAllowedWords = () => {
         : "";
 
     for (let i = 0; i < numTagItems; i++) {
-      tagItems += `<div class="tag-item">${allowedWordsData[i]}</div>`;
+      tagItems += `<div class="tag-item tag-text">${allowedWordsData[i]}</div>`;
     }
 
-    tagList.innerHTML = tagItems + moreResults;
+    tagList.innerHTML = tagItems + moreResults;    
```

Comment:
Can you revert this line? This PR should only affect only the lines that need to be modified. This will reduce the changes the reviewer needs to review and avoid unnecessary merge conflicts.